recipe/meta.yaml

```
@@ -22,14 +22,14 @@ requirements:
     - pip
     - setuptools_scm
     - numpy >=1.16
-    - pyarrow ==0.15.0.*
+    - pyarrow ==0.15.*
```

The preferred notation is
```suggestion
    - pyarrow 0.15.*
```